content/en/tracing/llm_observability/api.md

```diff
@@ -0,0 +1,331 @@
+---
```

Comment:
I modeled this page as closely to Datadog's API docs as possible, but some formatting features are not available to me that are used there. I'm open to feedback on the way this is organized.